src/Symfony/Bridge/Twig/Extension/AssetMapperExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class AssetMapperExtension extends AbstractExtension
+{
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('asset_mapper_source', [AssetMapperRuntime::class, 'source'], ['is_safe' => ['all']]),
```

Comment:
I'm not sure about this `"is_safe" => ['all']` .. is this not a bit risky ?

Reply:
I duplicated what `source()` from Twig core does so.. not sure.